### ISSUE TITLE
Shuffle dataloader with validation set

### ISSUE THREAD

**Issue by Scienceseb** [sangmin-git/LMC-Memory#14]
Hi !

I just want to point something kind of special, so it seems that your code even if I specified shuffle=True, don't make the valid dataloader shuffle.

So the shuffle=True is inactive:
```
 valid_dataset = Corem(args.valid_data_dir, seq_len=args.short_len+args.out_len, train=False)
 validataloader = DataLoader(valid_dataset, batch_size=args.batch_size, shuffle=True, num_workers=args.workers, drop_last=False)
```

Thanks for your help, for some reasons I need a validation dataloader shuffle.

**Comment by sangmin-git** [comment]
`shuffle=True` also can work with `validloader`.
I just have confirmed that the data indexes are shuffled in case of `shuffle=True`.

Thanks
